### DISCUSSION TITLE
Notification test

### DISCUSSION THREAD

**Discussion by MarcusAseth** [z1dev/zkanji#41] [General]
> I opened the Discussions page here for non-bug related stuff! It shows me as subscribed, so hopefully I'll get notifications of that one, even if GitHub fails to do that regarding bug reports...

testing if you get notifications from this board, feel free to delete this post after :D

**Comment by z1dev** [comment]
Still nothing. I checked and I have every email notification turned on. I even enabled "web and mobile" now just in case it's buggy and you need to allow everything, but no... 👎

**Comment by MarcusAseth** [comment]
that's curious :/  Not even email notifications for subscribed topics, like this one? I have an email on your reply here on my inbox, so that's working (on my side at least)

**Reply by z1dev** [reply]
I might register a new mail address for zkanji on gmail. The one I have is a local service and it's as bad as it gets. But zkanji is already taken there (I tried!) so any suggetsions?

**Reply by MarcusAseth** [reply]
z1dev@gmail.com ?  :D
Edit: oh, is a mail only for zkanji? nevermind then :S
zkanji.dict@gmail.com  ? or something like that?

**Reply by z1dev** [reply]
I'm contemplating making the new gmail right now. I hate how I must give my name and other personal information there...

**Reply by z1dev** [reply]
...grumble grumble...

**Comment by z1dev** [comment]
Registered a new mail and hopefully I'll get notifications now. (I'll still need to check my mail there often, not sure this solves anything... 😢

**Reply by MarcusAseth** [reply]
I'm reading gmail has a function of automatic forwarding that should let you easily forward those emails to the preferred address :)

**Reply by z1dev** [reply]
At least the notification through this address works! I could forward the mails, but then I'd give more information to Google! (I'm paranoid 😱 )
